packages/mdc-ripple/_mixins.scss

```diff
@@ -156,6 +186,22 @@
   }
   // stylelint-enable at-rule-empty-line-before, block-closing-brace-newline-after
 
+  &.mdc-ripple-upgraded {
+    &#{$pseudo} {
+      top: 0;
+      left: 0;
+      width: var(--mdc-ripple-fg-size);
```

Comment:
Do we need to use the dual-property method for `.mdc-ripple-upgraded`? I think we left that off initially b/c we don't even upgrade the ripple if Custom Vars aren't supported by the browser. Or, should we just do it for consistency?